proto/s2s.go

```diff
@@ -51,6 +51,22 @@ type Request struct {
 	FinishedJobs int       `json:"finishedJobs"` // the number of finished jobs in the request
 }
 
+// SuspendedJobChain (SJC) represents the data required to reconstruct and resume a
+// running job chain in the Job Runner.
+type SuspendedJobChain struct {
+	// request the SJC is for - unique identifier
+	RequestId string    `json:"requestId"`
+	JobChain  *JobChain `json:"jobChain"`
+	// job.Id -> total number of times a job has been tried
+	TotalJobTries map[string]uint `json:"totalJobTries"`
+	// job.Id -> number of times a job was tried the last time it was run
+	LastRunJobTries map[string]uint `json:"lastRunJobTries"`
+	// Id of 1st job in sequence -> number of times sequence has been tried
+	SequenceTries map[string]uint `json:"sequenceTries"`
+	// number of jobs that have been run (each job counts only once, even if retried)
+	NumJobsRun uint `json:"numJobsRun"`
```

Comment:
Does this need to be stored, or can it be recalculated when the SJC is resumed by counting completed jobs?